src/devices/cpu/drcbearm64.cpp

```diff
@@ -1286,26 +1269,27 @@ void drcbe_arm64::op_hashjmp(a64::Assembler &a, const uml::instruction &inst)
 		}
 	}
 
-	call_arm(a, TEMP_REG1);
-
 	Label lab = a.newLabel();
+	a.adr(REG_PARAM1, lab);
+	a.br(TEMP_REG1);
+
 	a.bind(lab);
-	a.adr(SCRATCH_REG1, lab);
 	emit_str_mem(a, SCRATCH_REG1, &m_near.hashstacksave);
+
 	a.mov(SCRATCH_REG1, 1);
 	emit_str_mem(a, SCRATCH_REG1.w(), &m_near.calldepth);
```

Comment:
You missed one line of the change here – `m_near.hashstacksave` is now being set to an unpredictable value because the `adr` now stores the address in `REG_PARAM`, rather than `SCRATCH_REG`.  It should be:

```c++
	Label lab = a.newLabel();
	a.adr(REG_PARAM1, lab);
	a.br(TEMP_REG1);

	a.bind(lab);
	emit_str_mem(a, REG_PARAM1, &m_near.hashstacksave);

	a.mov(SCRATCH_REG1, 1);
	emit_str_mem(a, SCRATCH_REG1.w(), &m_near.calldepth);
```